GG/src/Font.cpp

```diff
@@ -1253,140 +1216,136 @@ namespace DebugOutput {
     }
 }
 
-void Font::FillTextElements(const std::string& text,
-                            bool ignore_tags,
-                            std::vector<boost::shared_ptr<TextElement> >& text_elements) const
+std::vector<boost::shared_ptr<Font::TextElement> > Font::ComputeTextElements(
+    const std::string& text,
+    const Flags<TextFormat>& format) const
 {
```

Comment:
General remark:  Breaking down the commit into several commits like 'detending function content', 'renaming function' and 'parameter/return value usage' would help significantly when reviewing a PR.  I wasn't sure that this is essentially the same function until I used `git difftool` and applied some vimdiff changes locally.